kytos/core/api_server.py

```diff
@@ -57,11 +60,24 @@ def __init__(self, listen='0.0.0.0', port=8181,
                 Middleware(CORSMiddleware, allow_origins=["*"]),
             ],
         )
+        api_threadpool_size = get_thread_pool_max_workers().get('api', 40)
```

Comment:
If you could also document with a comment why we ended up with 40 here by default when it's not set on kytos.conf that'd be great for future readers to understand.

Reply:
Right. Let's keep at 160 then, if this ever needed to be adjusted we can revisit this, either way, good thing that it's configurable, and this looks very reasonable as a default, so it's great.